### PR TITLE
using different networktype for ocp-on-osp on different days of week

### PR DESCRIPTION
**Pull request by amitsagtani97**
For openshift install on osp role.
Setting OPENSHIFT_NETWORK_TYPE=OpenshiftSDN for idd days of week
and OPENSHIFT_NETWORK_TYPE=kuryr for the even days of week

### CODE REVIEW

File: jjb/dynamic/scale-ci_install_osp.yml [hunk 1]
```diff
@@ -21,6 +21,18 @@
...
         for (( iter=1; iter<=$JOB_ITERATIONS; iter++ ))
         do
+        
+        # Get the day of week and set networkType as OpenshiftSDN on odd days and kuryr on even days.
+        DOW=$(date +%u)
+        rem=$(( $DOW % 2 ))
+        if [ $rem -eq 0 ]
+        then
+          echo "Using kuryr as Network Type"
+          export OPENSHIFT_NETWORK_TYPE=kuryr
```
Comment (masco) [inline jjb/dynamic/scale-ci_install_osp.yml:31]: s/kuryr/Kuryr

**Comment by masco** [comment]
if we use this pipeline manually to check something, it will change the network type and it might cause some confusion to the user. thoughts?

**Comment by amitsagtani97** [comment]
@masco For running this pipeline manually we have to comment these conditions manually in the configure section

**Comment by amitsagtani97** [comment]
@masco added support to enter networkType manually.
Now if we define networkType in properties file, it will use that. Else it will be defined based on the day of week.

File: jjb/dynamic/scale-ci_install_osp.yml [hunk 2]
```diff
@@ -21,6 +21,22 @@
...
+        
+        # Get the day of week and set networkType as OpenshiftSDN on odd days and kuryr on even days in case no netork is defined manually.
+        if [ $OPENSHIFT_NETWORK_TYPE == "" ]
+        then
+            DOW=$(date +%u)
+            rem=$(( $DOW % 2 ))
+            if [ $rem -eq 0 ]
+            then
+                echo "Using kuryr as Network Type"
+                export OPENSHIFT_NETWORK_TYPE=kuryr
```
Comment (masco) [inline jjb/dynamic/scale-ci_install_osp.yml:33]: the value 'kuryr' shoud be in camel case 'Kuryr' i.e K in caps

**Review by masco — COMMENTED** [review]
this is perfect. just change the kuryr value

File: jjb/dynamic/scale-ci_install_osp.yml [hunk 3]
```diff
@@ -21,6 +21,22 @@
         cp ci/ocp_install_vars.yml vars/shift_stack_vars.yaml
         for (( iter=1; iter<=$JOB_ITERATIONS; iter++ ))
         do
+        
+        # Get the day of week and set networkType as OpenshiftSDN on odd days and kuryr on even days in case no netork is defined manually.
```
Comment (masco) [inline jjb/dynamic/scale-ci_install_osp.yml:25]: nit: typo in the comment s/netork/network

**Review by masco — APPROVED** [review]
LGTM

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**